code/modules/networks/computer3/mainframe2/misc_terms.dm

```diff
@@ -1708,6 +1708,7 @@
 
 	New()
 		..()
+		src.AddComponent(/datum/component/obj_projectile_damage)
```

Comment:
Sanity check for those who know components better: they need to be removed in `disposing` or similar, right?

Reply:
I think that gets done automatically later on, no need to manually do it.